tinyslam_run.launch

```diff
@@ -14,6 +14,13 @@
     <param name="occupancy_estimator" value="const" />
     <param name="skip_exceeding_lsr_vals" value="false" />
 
+    <!-- Topics setup -->
+    <param name="laser_scan" value="/base_scan" />
```

Comment:
laser_scan topic name is set via the remap above. The laser_scan param should be removed.